compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Optics.hs

```diff
@@ -13,17 +15,24 @@ module DA.Daml.LF.Ast.Optics(
     _PRSelfModule,
     exprPartyLiteral,
     exprValueRef,
+    packageRefs,
     templateExpr
     ) where
 
 import Control.Lens
 import Control.Lens.Ast
 import Control.Lens.MonoTraversal
+import Data.Fixed (Fixed)
 import Data.Functor.Foldable (cata, embed)
+import Data.Int (Int32, Int64)
```

Comment:
I'd just import the whole module

Reply:
Sure. c24463d00375c9f393def242afcdeeb991d9e0fc